bot/exts/gurkan/gurkan_stats.py

```diff
@@ -0,0 +1,115 @@
+from typing import Union
+
+from bot.bot import Bot
+from bot.constants import Emojis
+from bot.utils.is_gurkan import gurkan_check, gurkan_rate
+from discord import Color, Embed, Member
+from discord.ext.commands import Cog, Context, command
+
+
+RATE_DICT = {
+    range(1, 10): "pathetic",
+    range(10, 30): "not bad",
+    range(30, 60): "good",
+    range(60, 80): "cool!",
+    range(80, 95): "so Epic!!",
+    range(95, 100): "just wow, superb!",
+}
+
+
+class GurkanStats(Cog):
+    """Commands for showing stats on the Gurkan server."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @command(
+        name="gurkancount",
+        aliases=(
+            "gc",
+            "gurkcount",
+        ),
+        brief="Get the count of people who are valid gurkans",
+        help="""gurkancount
+
+            Get the count of people who are valid gurkans.
+            """,
+    )
+    async def gurkan_count(self, ctx: Context) -> None:
+        """
+        Goes through a list of all the members and uses regex to check if the member is a gurkan.
+
+        Sends the count of total Gurkans in the server,\
+        and the percentage of the gurkans to the server members.
+        """
+        members = [i.display_name for i in ctx.guild.members]
+        gurkans = sum(gurkan_check(member) for member in members)
```

Comment:
Instead of constructing the list with list comprehension, would it be possible to use `map()` or the following :point_down: ?
```suggestion
        gurkans = sum(gurkan_check(member.display_name) for member in ctx.guild.members)
```